src/cli/index.ts

```diff
@@ -42,16 +42,33 @@ export default class Cli {
   private readonly argv: string[]
   private readonly cwd: string
   private readonly stdout: IFormatterStream
+  private readonly warn: (message: string) => void
 
-  constructor({ argv, cwd, stdout }) {
+  constructor({ argv, cwd, stdout, warn = console.warn }) {
```

Comment:
Could we pass in stderr and directly write to that (or create a [console](https://nodejs.org/api/console.html#console_new_console_stdout_stderr_ignoreerrors) given the passed in stdout / stderr)? I'd prefer that since we are already passing in stdout.

Reply:
Sure will look at that - I thought the `stdout` here was the output stream for the formatter

Reply:
Yeah, stdout is passed in for the formatter. The only reason to create our own console is to be able to still make use of the same API we were already using

Reply:
Done - creating a console from the stdout we pass in

Reply:
I was thinking we also pass in stderr and pass that into the console. The warning should definitely go to stderr. Is it currently going to stdout?

Reply:
Sorry yeah - going to `stderr` now